ui/plugins/eu.esdihumboldt.hale.ui/src/eu/esdihumboldt/hale/ui/service/schema/internal/SchemaServiceImpl.java

```diff
@@ -71,6 +71,22 @@ public SchemaServiceImpl(ProjectService projectService) {
 		super(projectService);
 	}
 
+	/**
+	 * @see eu.esdihumboldt.hale.ui.service.schema.SchemaService#getSchemas(java.lang.String)
+	 */
+	@Override
+	public Schema getSchemas(String resourceId, SchemaSpaceID spaceID) {
```

Comment:
`getSchemas` is a bit misleading as a name I think, I would suggest `getSchema` instead.

Reply:
Code refactored as suggested.